### ISSUE TITLE
Cannot register my routes

### ISSUE THREAD

**Issue by scorreia** [dmossakowski/additive-spotify-analyzer#32]
I can register to ESC 15 competition, but after I select my routes and click on "save", then the site tells me "You don't have a permission to do that"

**Comment by dmossakowski** [comment]
Yes, I think you're only permissions for competitions 'abc' and 'def'
